src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,188 @@
+                if (approveOperation()) {
+                    String[] pathAndFileName = handleInputPathFilename();
+                    createFile(pathAndFileName[0], pathAndFileName[1], inputString);
+                }
+        }
+    }
+
+    private String[] handleInputPathFilename() {
+        System.out.println(ENTER_PATH_END_FILENAME);
+        String[] pathFilename = scanner.nextLine()
+                .replaceAll("\\s+", " ").trim().split(" ");
+        if (pathFilename.length != 2) {
+            System.out.println(String.format(INCORRECT_PATH_OR_FILENAME, pathFilename[0]));
+            handleInputPathFilename();
+        }
+        return pathFilename;
+    }
+
+    private boolean approveOperation() {
+        String input = scanner.nextLine().trim();
+        if (!input.toLowerCase().equals("y") && !input.toLowerCase().equals("n")) {
+            System.out.println(String.format(UNKNOWN_COMMAND, input));
+        }
+        return input.equals("y");
+    }
+
+    private void readFileInfo(String path, String fileName) {
+        if (!path.endsWith(File.separator)) {
+            path += File.separator;
+        }
+        Path fileNamePath = Paths.get(path + fileName);
+        try {
+            Date date = new Date(Files.getLastModifiedTime(fileNamePath).toMillis());
+            System.out.println(String.format(
+                    "Symbols count : %s", Files.readString(fileNamePath).length()));
+            System.out.println(String.format(
+                    "Rows count  : %s", Files.readAllLines(fileNamePath).size()));
+            System.out.println(String.format(
+                    "Last modified : %s", date.toString()));
+            System.out.println(String.format(
+                    "File size : %s bytes", Files.size(fileNamePath)));
+        } catch (IOException e) {
+            System.out.println(FILE_NOT_FOUND);
+        }
+    }
+
+    private void readFile(String path, String fileName) {
+        if (!path.endsWith(File.separator)) {
+            path += File.separator;
+        }
+        Path fileNamePath = Paths.get(path + fileName);
+        String line;
+        try (BufferedReader br = Files.newBufferedReader(fileNamePath)) {
+            while ((line = br.readLine()) != null) {
+                System.out.println(line);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+            System.out.println(FILE_NOT_FOUND);
+        }
+    }
+
+    private void createFile(String path, String fileName, String content) {
+        if (!path.endsWith(File.separator)) {
+            path += File.separator;
+        }
+        Path fileNamePath = Paths.get(path + fileName);
+        Path directoriesPath = Paths.get(path);
+        try {
+            if (Files.notExists(directoriesPath)) {
+                Files.createDirectories(directoriesPath);
+            }
+            Files.createFile(fileNamePath);
+        } catch (FileAlreadyExistsException e) {
```

Comment:
don't build logic on try catch blocks use if else instead( but not too much, keep your code understandable)

Reply:
done